src/services/media/index.ts

```diff
@@ -57,6 +57,10 @@ export class MediaPlayerService {
 
         this.mediaTitle.set(noMediaText.get());
 
+        for (const player of this._mprisService.players) {
```

Comment:
```suggestion
        for (const player of this._mprisService.get_players()) {
```
For list type properties let's use getter methods due to some issue with Vala and GJS: https://aylur.github.io/astal/guide/typescript/faq#error-can-t-convert-non-null-pointer-to-js-value

Reply:
I'll keep that in mind for next time!

Thanks!